tools/challenge-auditor/index.ts

```diff
@@ -26,9 +35,32 @@ const superBlockFolderMap = {
 // the audit, we just ignore them.
 const blocksThatNeedToMove = ['d3-dashboard'];
 
+// Adding types for getChallengesForLang is possible, but not worth the effort
+// at this time.
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
![TearsGIF](https://user-images.githubusercontent.com/88248797/182533670-4c0d3981-599a-402f-a4ba-61560b028582.gif)

Reply:
I can actually post gifs here🤣

Reply:
I wouldn't worry about it @Sboonny. We would have to declare the types or migrate getChallenges.js.  The former is a maintenance headache (we'll likely change getChallenges in the not-to-distant future) and the latter is *hard*.